src/applications/mhv-supply-reordering/components/IneligibleErrorMessage.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { format } from 'date-fns';
+import DlcServiceTelephone from './DlcSupportTelephone';
+
+/**
+ * Generates an alert for a veteran that is not eligible to order supplies.
+ * @param {string} nextAvailabilityDate the date supplies will be available to reorder
+ * @returns the alert
+ */
+const IneligibleErrorMessage = ({ nextAvailabilityDate }) => {
+  const formattedDate = format(new Date(nextAvailabilityDate), 'MMMM d, yyyy');
```

Comment:
Does this need to handle the possibility of the string not being parseable as a date?